### ISSUE TITLE
macOS 13 runner needs Xcode UI testing capabilities

### ISSUE THREAD

**Issue by asif-marakkar** [actions/runner-images#8611]
### Description

I'm seeing an issue on macos-13 runner. 
The job fails with error "UITests-Runner (14751) encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted. (Underlying Error: Test crashed with signal kill before starting test execution.))".
I moved from macOS 12 to macOS 13 runner and started seeing this.
Operating System: **macOS 13.6 - 22G120**
Runner Image: **macos-13 Version: 20231002.1**
Xcode 15 and iOS 17.0

### Platforms affected

- [ ] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [X] macOS 13
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

![Screenshot 2023-10-19 at 8 26 27 PM](https://github.com/actions/runner-images/assets/56037274/f51f3a81-fb33-47c4-85a4-5cb5cce1c76e)

### Is it regression?

yes

### Expected behavior

Xcode UI tests should work

### Actual behavior

They don't work

### Repro steps

Xcode UI testing is failing on macos-13. We have been asked to run the pipeline Xcode 15 and OS version 17.0.

**Comment by asif-marakkar** [comment]
Let me know if you need additional information.

**Comment by mikhailkoliada** [comment]
Duplicate of https://github.com/actions/runner-images/issues/8546
